src/tracy.h

```diff
@@ -111,33 +111,33 @@ inline void __set_error_msg(void) {}
 
 
 /* Add current position to traceback and return `err`. */
-#define RETURN_ERROR(err) do { \
-  __add_error_trace(__FILE__, __FUNCTION__, __LINE__); \
+#define TRC_RETURN_ERROR(err) do { \
+  trc_private_add_error_trace(__FILE__, __FUNCTION__, __LINE__); \
   return (err); \
 } while (0)
 
 
-/* Like `RETURN_ERROR`, but only if and only if `predicate` is true. */
-#define RETURN_ERROR_IF(predicate, err) do { \
+/* Like `TRC_RETURN_ERROR`, but only if and only if `predicate` is true. */
+#define TRC_RETURN_ERROR_IF(predicate, err) do { \
   if ((predicate)) { \
-    RETURN_ERROR(err); \
+    TRC_RETURN_ERROR(err); \
   } \
 } while (0)
 
 
-/* Like `RETURN_ERROR`, but only if `err` != OK. */
-#define RETURN_ON_ERROR(err) do { \
-  err_t const __ror_local_err = (err); \
-  RETURN_ERROR_IF(__ror_local_err != OK, __ror_local_err); \
+/* Like `TRC_RETURN_ERROR`, but only if `err` != TRC_OK. */
+#define TRC_RETURN_ON_ERROR(err) do { \
+  TRC_err const trc_private_ror_local_err = (err); \
+  TRC_RETURN_ERROR_IF(trc_private_ror_local_err != TRC_OK, trc_private_ror_local_err); \
```

Comment:
Line exceeds 80 cols.
Your search-and-replace caused this to happen in many places, fix them too.